### PR TITLE
DRIV-6546 and DRIV-6545

### PR DESCRIPTION
**Pull request by MJovanovicC4**
Features:

- Updating `modified` date
- Passing an argument with the driver version to update the `version` tag into

### CODE REVIEW

**Review by WWalshC4 — APPROVED** [review]
I like these improvements, and I really like the use of named flagged arguments vs positional args.

File: dp3/driverpackager.py [hunk 1]
```diff
@@ -502,6 +555,10 @@ def main():
                         help="[optional] Unzip the c4z in the target location.")
     parser.add_argument("-ae", "--allowexecute", action="store_true",
                         help="[optional] Allow Execute in Lua Command window.")
+    parser.add_argument("-um", "--updatemodified", action="store_true",
```
Comment (FDrazicC4) [inline dp3/driverpackager.py:558]: I think that we should use --update-modified.

File: dp3/driverpackager.py [hunk 2]
```diff
@@ -502,6 +555,10 @@ def main():
...
+                        help="[optional] Update driver modified date.")
+    parser.add_argument("-uv", "--updateversion", nargs=1,
```
Comment (FDrazicC4) [inline dp3/driverpackager.py:560]: I think that we should use --update-version.

**Review by FDrazicC4 — CHANGES_REQUESTED** [review]
Let's also update the GitHub action and its documentation.

**Review by FDrazicC4 — APPROVED** [review]

**Pull request merged**